dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ExcludedErrors.java

```diff
@@ -36,8 +42,12 @@ static ExcludedErrors create() throws IOException {
       ImmutableList<LinkageErrorMatcher> defaultMatchers =
           ExclusionFileParser.parse(defaultRuleUrl);
       exclusionMatchers.addAll(defaultMatchers);
+
+      if (exclusionFile != null) {
+        exclusionMatchers.addAll(ExclusionFileParser.parse(exclusionFile));
+      }
     } catch (SAXException | VerifierConfigurationException ex) {
-      throw new IOException("Could not read default exclusion rule", ex);
+      throw new IOException("Could not read exclusion rule", ex);
```

Comment:
rule --> rule file.

Reply:
This big try-catch is split  into two: one for default exclusion rule and the other for the argument.